Dockerfile.fips

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10-1154 AS base
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5-1736404155 AS base
```

Comment:
In talking to compliance only 9.2 is officially FIPS approved, may need to bump this down to that tlll further notice.

Reply:
Marking as changes requested

Reply:
To follow up:
* The RHEL 9 openssl module for 9.4+ has been in process or certified which is what we leverage with our FIPS build
* The Kernel under 4.14 clusters is RHEL 8, so the kernel level is covered for containers running UBI 9 on top of a RHEL 8 host (4.15 or lower OCP)
Therefore any 9.x image should be ok to use. This is good to merge as is